Tests/PackageLoadingTests/PackageBuilderTests.swift

```diff
@@ -1376,6 +1376,7 @@ class PackageBuilderTests: XCTestCase {
             "ios": "8.0",
             "tvos": "9.0",
             "watchos": "2.0",
+            "android": "7.0",
```

Comment:
Remove

Reply:
Had to set to 0.0 just like linux above for this test to pass.